src/bin/nydus-image/merge.rs

```diff
@@ -166,6 +171,10 @@ impl Merger {
             }
         }
 
+        assert!(fs_versions.len() == 1);
```

Comment:
How about recording the first fs version in the `for` loop? so that we can throw a error like this:

`bail!("inconsistent fs version between layers, expected version {}", expected);`

Reply:
fixed